requirements-base.txt

```diff
@@ -22,12 +22,12 @@ google-cloud-pubsub==2.2.0
 google-cloud-storage==1.35.0
 jsonschema==3.2.0
 lxml==4.6.3
-maxminddb==1.5.4
+maxminddb==2.0.3
```

Comment:
The [changes](https://github.com/maxmind/MaxMind-DB-Reader-python/blob/main/HISTORY.rst) are pretty trivial.